Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -138,3 +138,32 @@ The widget allows you to set up a few things as mentioned below:
 - Buttons
 
 ![movePage.jpg](images/The-Latest-Form-Feed-Widget1.png)
+
+
+### The Map Widget
+
+![movePage.jpg](images/The-Map-Widget1.png)
+
+The Map Widget is a feature that lets you use google maps to show the location of your choice, this could be if you need to show where your company is located.
```

Comment:
```suggestion
The Map Widget is a feature that lets you use Google Maps to show the location of your choice, this could be if you need to show where your company is located.
```